support-modules/acquisition-events/src/main/scala/com/gu/support/acquisitions/acquisitions-stream/AcquisitionsStreamService.scala

```diff
@@ -25,7 +25,13 @@ sealed trait AcquisitionsStreamServiceConfig {
 case class AcquisitionsStreamEc2OrLocalConfig(val streamName: String, val credentialsProvider: AWSCredentialsProviderChain) extends AcquisitionsStreamServiceConfig
 case class AcquisitionsStreamLambdaConfig(val streamName: String) extends AcquisitionsStreamServiceConfig
 
-class AcquisitionsStreamService(config: AcquisitionsStreamServiceConfig, region: String = "eu-west-1") {
+trait AcquisitionsStreamService {
```

Comment:
I made it a trait so that we can define a mock implementation, which I think is preferable to using mockito